src/bluesky/tests/test_new_examples.py

```diff
@@ -697,13 +697,15 @@ def test_trigger_and_read(hw):
     for msg in msgs:
         msg.kwargs.pop("group", None)
         msg.kwargs.pop("timeout", None)
+        msg.kwargs.pop("move_on", None)
```

Comment:
Can we add this to the expected rather than popping here?  In the case of `group` it is random so we can not write down what it _should_ be, but `move_on` is a bool so we should be able to!  We probably should be checking that the group is internally self-consistent (the blame on that goes back to 2016).

I think it is fair to ask for `move_on` to be handled better, but bonus points for fixing timeout and group as well!